apps/bare-expo/package.json

```diff
@@ -68,7 +68,7 @@
     "@react-native-masked-view/masked-view": "0.2.8",
     "@react-native-picker/picker": "2.4.8",
     "@react-native-segmented-control/segmented-control": "2.4.0",
-    "@shopify/flash-list": "1.3.1",
+    "@shopify/flash-list": "1.4.0",
```

Comment:
Hmm, this should also make a change in Podfile.lock, but it didn't, any idea why? 🤔

Reply:
It made a change in Podfile: https://github.com/expo/expo/blob/5b244e601b916adba733f389a7b5e353075b1d93/ios/Podfile.lock#L2097 ;)

Reply:
But not in bare-expo 😅

Reply:
Fixed ;)

Reply:
Haha, it seems it's still not fixed 😅 Now it downgrades gesture handler and still doesn't bump FlashList version 🤔

Reply:
Sorry, my bad. Now everything should be ok ;)